internal/utils/helper.go

```diff
@@ -33,3 +34,32 @@ func IsDirectory(name string) bool {
 	}
 	return info.IsDir()
 }
+
+func AssertOutputIsEqual(expected, actual []byte, verbose bool) error {
+	if string(expected) != string(actual) {
+		if verbose {
+			log.Printf("expected.go:\n%s\n", string(expected))
+			log.Printf("generated file:\n%s\n", string(actual))
+		}
+		return fmt.Errorf("expected.go and generated file are different")
+	}
+	return nil
+}
+
+// RunCommand runs a command and returns an error if it fails.
+// If verbose is true, the command output is printed.
+func RunCommand(command []string, verbose bool) error {
```

Comment:
Does this return an error in case of non-zero exit code ?
can we test it ?

Reply:
EDIT: hard to test considering is depends on the OS, ignore

Reply:
Yes. It's the behavior of the `Run()` method.

Reply:
https://pkg.go.dev/os/exec#ExitError